packages/permissions/README.md

```diff
@@ -6,9 +6,10 @@
 
 React components to declaratively handle MC permissions.
 
-- `branchOnPermissions()`
-- `<RestrictedByPermissions>`
+- `useIsAuthorized()`
```

Comment:
I added this and the documentation for it (it was missing). The rest is just reordering